content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation](/tracing/trace_collection/) is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation](/tracing/trace_collection/automatic_instrumentation/) is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries](https://opentelemetry.io/docs/concepts/instrumentation/libraries/) from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
+
+## Language support
+
+| Language | Minimum version required |
+|----------|--------------------------|
+| Java     | v1.35.0                  |
+| Go       |                          |
+| Python   |                          |
+| Ruby     |                          |
+| Node.js  |                          |
+| PHP      |                          |
+| .NET     |                          |
```

Comment:
A bit nit picky, but can we put a placeholder here like `TBA`?

Reply:
We can - but I'd hoped to have this all filled in before this was published

If we want to publish this for selected languages first then I can add `TBA` here and comment out their tabs...

Reply:
I've added TBA for the missing versions